src/routes/index.js

```diff
@@ -43,4 +45,36 @@ const router = Router();
  */
 router.post('/users', createUser);
 
+
+
+/**
+ * @swagger
+ *
+ * /api/v1/users:
+ *   post:
+ *     description: User Registration Endpoint
+ *     produces:
+ *       - application/json
+ *     request:
+ *         content:
+ *         - application/json
+ *         schema:
+ *           type: array
+ *           items:
+ *         $ref: '#/definitions/users'
+ *     responses:
+ *       201:
+ *         description: User created
+ *       400:
+ *         description: Bad request.
+ *       401:
+ *         description: Authorization information is missing or invalid.
+ *       404:
+ *        description: A user with the specified ID was not found.
+ *       5XX:
+ *        description: Unexpected error.
+ */
+router.post('/forgot-password', forgotPassword);
```

Comment:
we can prefix this endpoint with `/users/`.

Reply:
Okay, I will do this